backends/qualcomm/passes/convert_addmm_back_to_linear.py

```diff
@@ -101,6 +115,26 @@ def _convert_addmm(self, gm: torch.fx.GraphModule, src_partition: SourcePartitio
             for user in addmm_node.users.copy():
                 user.replace_input_with(addmm_node, linear_node)
 
+    def _convert_bmm(self, gm: torch.fx.GraphModule, src_partition: SourcePartition):
+        output = src_partition.output_nodes[0]
+        bmm_node = [n for n in src_partition.nodes if n.target == self.bmm][0]
+        add_node = [n for n in src_partition.nodes if n.target == self.add][0]
+
+        # weight -> expand_copy -> view_copy -> input of bmm
+        weight_node = bmm_node.args[1].args[0].args[0].args[0]
```

Comment:
How do you ensure that you actually do see this pattern?

Reply:
Thank you for bringing this to our attention. The pattern you mentioned was found in VIT. However, we are still investigating how to check all patterns for linear. If you have any recommendations, I would be happy to hear them.